Week1/homework/app.js

```diff
@@ -1,11 +1,186 @@
 'use strict';
+
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+
+  //2.1
+  const arrayBookTitles = [
+
+    'life_of_faith',
+    'the_sign_and_seal',
+    'the_book_of_genesius',
+    'the_book_of_reveletion',
+    'how_to_relate_with_children',
+    'what_is_man',
+    'even_the_stones_are_burning',
+    'life_of_hope',
+    'computer_architecture_and_organization',
+    'maximize_your_social_media'
   ];
 
+  //2.2
+  //console.log(arrayBookTitles);
+
+  //2.3
+
+  function bookTitles() {
+
+    const listcontainer = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(listcontainer);
+    const ul = document.createElement('ul');
+    ul.setAttribute('type', 'square');
+    ul.setAttribute('id', 'booktitle');
+    listcontainer.appendChild(ul);
+
+    for (let i = 0; i < arrayBookTitles.length; i++) {
+
+      let li = document.createElement('li');
+      li.innerHTML = arrayBookTitles[i];
+      ul.appendChild(li);
+
+    }
+  }
+
+  //2.4
+
+  const objBooksInfo = {
+    life_of_faith: {
+      title: 'Life of faith',
+      language: 'English',
+      author: 'Pop Sheonoda'
+    },
+    the_sign_and_seal: {
+      title: 'The sign and seal',
+      language: 'English',
+      author: 'Graham Hancock'
+    },
+    the_book_of_genesius: {
+      title: 'The book of Genesius',
+      language: 'Tigrigna',
+      author: 'prophet Moses'
+    },
+    the_book_of_reveletion: {
+      title: 'The book of Reveletion',
+      language: 'Geez',
+      author: 'St.John'
+    },
+    how_to_relate_with_children: {
+      title: 'How to relate with children',
+      language: 'English',
+      author: 'Pop Sheonoda'
+    },
+    what_is_man: {
+      title: 'What is man',
+      language: 'English',
+      author: 'Pop Sheonoda'
+    },
+    even_the_stones_are_burning: {
+      title: 'Even the stones are burning',
+      language: 'English',
+      author: 'Roy Patheman'
+    },
+    life_of_hope: {
+      title: 'Life of hope',
+      language: 'English',
+      author: 'Pop Sheonoda'
+    },
+    computer_architecture_and_organization: {
+      title: 'Computer architecture and organization',
+      language: 'English',
+      author: 'Kanj Sharma'
+    },
+    maximize_your_social_media: {
+      title: 'Maximize your social media',
+      language: 'English',
+      author: 'Neal Sheffar'
+    }
+
+  }
+
+  //2.5
+
+
+  function booksInfo() {
+
+    for (let key in objBooksInfo) {
+      if (objBooksInfo.hasOwnProperty(key)) {
+        let book = objBooksInfo[key];
+        let booksContainer = document.createElement('div');
```

Comment:
Rather than creating a new `div` to display this information you could use `getElementById()` to get a `li` you created earlier for each book and add the book details to that `li`.